pkg/config/default_linux.go

```diff
@@ -17,3 +20,16 @@ func isCgroup2UnifiedMode() (isUnified bool, isUnifiedErr error) {
 	}
 	return
 }
+
+// getDefaultProcessLimits returns the nofile and nproc for the currentprocess in ulimits format
+func getDefaultProcessLimits() []string {
+	rlim := unix.Rlimit{Cur: 1048576, Max: 1048576}
```

Comment:
Can we make these two values string constans and add comments explaining why we chose these values?

Reply:
Sure except I have no idea where they came from...

Reply:
Looks like the limit meant something in RHEL5.

https://stackoverflow.com/questions/1212925/on-linux-set-maximum-open-files-to-unlimited-possible/1213069#1213069

Comes from Docker.

I think we can set this to max_proc.